packages/ember-testing/lib/helpers/fill_in.js

```diff
@@ -32,7 +32,7 @@ export default function fillIn(app, selector, contextOrText, text) {
   el = $el[0];
   focus(el);
 
-  $el.eq(0).val(text);
+  el.value = text;
```

Comment:
Can you confirm if `$el.eq(0).val(text)` supported content editable elements? I don't recall if it did, but if it did then this would be a regression (since when content editable set you need to set `innerHTML`).

See [related code](https://github.com/emberjs/ember-test-helpers/blob/master/addon-test-support/@ember/test-helpers/dom/fill-in.js#L39-L43) from `@ember/test-helper`.

Reply:
Seems that both jquery and JS.  And yeah def need to set that innerHTML! :+1:  

```
el = document.createElement('div')
$(el).attr('contenteditable', 'true')
$(el).val('wat')
$(el).text() // or $(el).html() // ""
```